src/index.js

```diff
@@ -0,0 +1,80 @@
+import Icon from './three-dots-vertical.svg';
+import Arrow from './arrow-return-left.svg';
+import Refresh from './arrow-repeat.svg';
+import './style.css';
+
+const tasksList = [];
+
+class Task {
+  constructor(description, index, completed = false) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+const printTask = (description) => {
+  const container = document.querySelector('ul');
+  const task = document.createElement('li');
+  task.classList.add('tasks');
+  const check = document.createElement('input');
+  const text = document.createElement('label');
+  const icon = new Image();
+  icon.src = Icon;
+  check.type = 'checkbox';
+  text.textContent = description;
+  task.appendChild(check);
+  task.appendChild(text);
+  task.appendChild(icon);
+  container.appendChild(task);
+};
```

Comment:
- [ ] Populate your taskList with examples. And remember to use the 3 dots icon.

Your app:

![imagen](https://user-images.githubusercontent.com/60631456/126581018-b364c77f-b4fc-4c33-8975-fb7fbdb44dc5.png)

Target app:

![imagen](https://user-images.githubusercontent.com/60631456/126581054-ef8b3494-11ff-4e19-b805-3a3fbb191a3b.png)

Reply:
Well addressed this!